### DISCUSSION TITLE
brandAlternative

### DISCUSSION THREAD

**Discussion by AnaMontes11** [Telefonica/mistica-design#562] [Proposals]
Con la llegada de la nueva marca Movistar se añade al azul de brand actual, otro azul brand más oscuro pero con gran relevancia siendo usado en textos y backgrounds por lo que tenemos que valorar cómo añadir este nuevo color a las constantes. 
Una solución podría ser crear un brandAlternative.

<img width="820" alt="Captura de pantalla 2022-06-28 a las 17 21 29" src="https://user-images.githubusercontent.com/88187691/176657353-c35a5502-6cad-4e24-880b-10cf79d9effd.png">

**Comment by yceballost** [comment]
En mi cabeza el planteamiento de tener los siguientes estilos funciona:
- backgroundBrandAlternative
- brandAlternative

En el resto de marcas que no quieran tener un color alternativo de marca estos colores serán igual al color de marca

Deberíamos tener los siguientes también?
- brandAlternativeLow
- brandAlternativeHigh

**Reply by HectorSancho** [reply]
Pues probablemente sean necesarios para dividers y estados de elementos sobre ese color. Además de otras constantes que se utilicen en otros componentes (ejem. navigationBarBackground)

**Reply by AnaMontes11** [reply]
Por coherencia con lo planteado para el brand/brandLow/brandHigh al igual que los neutrals, creo que deberían existir

**Comment by yceballost** [comment]
Añado además que componentes como el header tendrá que evolucionarse para admitir el uso de este `backgroundBrandAlternative` de fondo

**Comment by yceballost** [comment]
Hablado con el equipo de marca sobre el uso de color azul oscuro vs azul claro

Sustituir en la marca Movistar el azul claro por el azul oscuro, y usar el azul claro para cosas puntuales (por ejemplo, yo veo que toda la parte de interacción, botones, controles, etc, podría ir en azul claro)

Se puede seguir usando el azul claro para masas, pero cosas muy puntuales, yo me imagino por ejemplo una highlighted card que pueda ir en este azul claro.

Esto cambia un poco el planteamiento inicial, habrá que cambiar el backgroundBrand al azul oscuro, y crear los brandAlternative con el azul claro.

Han actualizado el BF: https://brandfactory.telefonica.com/document/2372#/nuestra-identidad/layout

## Priority
Se está trabajando sobre ello, pero a priori no preocupa en que las dos tipología de usuarios se unifiquen

**Reply by AnaMontes11** [reply]
Te han dicho algo sobre el uso del azul oscuro en textos?

**Reply by yceballost** [reply]
no he llegado a ese detalle, han sido 5 min de reunión 😁

**Reply by yceballost** [reply]
parece que en brand factory usan el azul oscuro en los textos

**Reply by AnaMontes11** [reply]
Si, confirmado
